src/actions/index.js

```diff
@@ -62,10 +63,12 @@ export default config => {
 
     return dispatch({ type: 'remove', path, previous })
   }
+  let toJsIsEqual =(x,y)=>_.isEqual(toJS(x),y)
```

Comment:
I need some advice on this function.  most of the time of this PR was spend on how to design the compare function of snapshot previous and the value for mutating. 
I use this function is used by _.isMatchWith in line 71. 
The original approach is F.simpleDiff then I switch to _.match. both of them do not work because of the array value. 
for example x={a:[],b:1},y= {a:[1]},   _.match(y,x) will return true and the F.simpleDiff will return {}.

Then I changed the function to  _.isMatchWith(_.isEqual, value,previous) however, it failed on ContextureMobx test. because the _.isEqual could not tell observable array [] and [] are same. 

I think toJsIsEqual is the best option for us because we always use it with ContextureMobx

We could also add a  compare function arg to  mutate function, it will change to 
```javascript
 mutate = _.curry(async (path, value, isForceUpdate = false,compareFn=_.isEqual)
```